### PR TITLE
Add IntroScene with skip intro btn

### PR DESCRIPTION
**Pull request by pietrzakacper**

### CODE REVIEW

File: src/scenes/IntroScene.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      720 - 32,
+      'Skip intro',
+      {
+        originX: 1,
+        originY: 1,
+      }
+    );
+    skipIntroButton.on('click', () => {
+      this.sound.stopAll();
+      this.scene.start('MainMenuScene', { skipAnimation: true });
```
Comment (bibixx) [inline src/scenes/IntroScene.ts:23]: Why do we need `{ skipAnimation: true }`

File: src/scenes/LoadingScene.ts [hunk 2]
```diff
@@ -92,6 +92,10 @@ export class LoadingScene extends Phaser.Scene {
   };
 
   private changeScene = () => {
-    this.scene.start('MainMenuScene');
+    if (shouldSkipIntro()) {
```
Comment (bibixx) [inline src/scenes/LoadingScene.ts:95]: Let's add another env variable for that intro. When I'll be developing the intro I want to skip the bodging bear intro, but not the history intro (which I'm developing 😉)

**Review by bibixx — CHANGES_REQUESTED** [review]

File: src/scenes/LoadingScene.ts (hunk 2 above)
Comment (morgana54) [reply src/scenes/LoadingScene.ts:95]: ;)

Comment (morgana54) [reply src/scenes/LoadingScene.ts:95]: what a shitty PR

**Review by morgana54 — APPROVED** [review]

**Pull request closed without merging**